dev-games/trenchbroom/patches/trenchbroom-2025.1.patchset

```diff
@@ -0,0 +1,9675 @@
+Date: Mon, 17 Feb 2025 01:57:41 -0600
+Subject: [PATCH 01/12] CMakeLists.txt: remove vcpkg
+
+---
+ CMakeLists.txt | 3 ---
+ 1 file changed, 3 deletions(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 81b4e73a1..87f2a112f 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -16,9 +16,6 @@ if(POLICY CMP0063)
+   cmake_policy(SET CMP0063 NEW)
+ endif()
+ 
+-set(CMAKE_TOOLCHAIN_FILE "${CMAKE_SOURCE_DIR}/vcpkg/scripts/buildsystems/vcpkg.cmake")
+-set(VCPKG_OVERLAY_PORTS "${CMAKE_SOURCE_DIR}/vcpkg-overlay-ports/freeimage;${CMAKE_SOURCE_DIR}/vcpkg-overlay-ports/openexr")
+-
+ set(CMAKE_C_VISIBILITY_PRESET hidden)
+ set(CMAKE_CXX_VISIBILITY_PRESET hidden)
+ set(CMAKE_VISIBILITY_INLINES_HIDDEN TRUE)
+-- 
+2.48.1
+
+
+From e0967bfc5506811266605e059b4306ddf0740a00 Mon Sep 17 00:00:00 2001
+From: erysdren <contact@erysdren.me>
+Date: Mon, 17 Feb 2025 02:04:19 -0600
+Subject: [PATCH 02/12] CMakeLists.txt: remove pandoc
+
+---
+ CMakeLists.txt | 11 -----------
+ 1 file changed, 11 deletions(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 87f2a112f..0e2afc59f 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -91,17 +91,6 @@ if(NOT GIT_FOUND)
+   message(FATAL_ERROR "Could not find git")
+ endif()
+ 
+-# Find Pandoc
+-if(NOT PANDOC_PATH)
+-  find_program(PANDOC_PATH NAMES "pandoc" DOC "Pandoc program location")
+-
+-  if(PANDOC_PATH STREQUAL "PANDOC_PATH-NOTFOUND")
+-    message(FATAL_ERROR "Could not find pandoc")
+-  else()
+-    message(STATUS "Found Pandoc: ${PANDOC_PATH}")
+-  endif()
+-endif()
+-
+ # Prevent overriding the parent project's compiler/linker
+ # settings on Windows
+ set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
+-- 
+2.48.1
+
+
+From 145ce49ab1b012a34ff23c733eae79d1f2237ba3 Mon Sep 17 00:00:00 2001
+From: erysdren <contact@erysdren.me>
+Date: Mon, 17 Feb 2025 02:12:06 -0600
+Subject: [PATCH 03/12] freetype-config.cmake: add haiku paths
+
+---
+ cmake/packages/freetype-config.cmake | 6 ++++--
+ 1 file changed, 4 insertions(+), 2 deletions(-)
+
+diff --git a/cmake/packages/freetype-config.cmake b/cmake/packages/freetype-config.cmake
+index ccacae6c9..54113e2ed 100644
+--- a/cmake/packages/freetype-config.cmake
++++ b/cmake/packages/freetype-config.cmake
+@@ -14,7 +14,8 @@ find_path(freetype_INCLUDE_DIR
+   /usr/include/freetype2
+   /usr/local/include
+   /sw/include
+-  /opt/local/include)
++  /opt/local/include
++  /boot/system/develop/headers/freetype2)
+ find_library(freetype_LIBRARY
+   NAMES FreeType freetype
+   PATHS
+@@ -23,7 +24,8 @@ find_library(freetype_LIBRARY
+   /usr/local/lib64
+   /usr/local/lib
+   /sw/lib
+-  /opt/local/lib)
++  /opt/local/lib
++  /boot/system/develop/lib)
+ 
+ include(FindPackageHandleStandardArgs)
+ find_package_handle_standard_args(freetype
+-- 
+2.48.1
+
+
+From cc29a767070520abd79694373b581737a538c3d1 Mon Sep 17 00:00:00 2001
+From: erysdren <contact@erysdren.me>
+Date: Mon, 17 Feb 2025 02:18:20 -0600
+Subject: [PATCH 04/12] CMakeLists.txt: use vendored miniz
```

Comment:
Ideally you would use a second source for https://github.com/richgel999/miniz and link it into the source tree. Having a smaller patchset is preferred.